app/admin/user.rb

```diff
@@ -6,6 +6,23 @@
     user.send_request_to_patron_creator_service if user.valid? # if not valid, errors will show on the form
   end
 
+  csv do
+    column :id
+    column :email
+    column :current_sign_in_at
+    column :last_sign_in_at
+    column :created_at
+    column :updated_at
+    column :barcode
+    column :first_name
+    column :last_name
+    column :alt_email
+    column :alt_barcodes
+    column(:school_name) { |user| user.school.name}
```

Comment:
In case a user doesn't have a school for some reason, so these three lines could look like this:

`{ |user| user.school.present? ? user.school.name : '' }`